src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Bulldozer(), new Truck(), new Excavator()};
 
+        for (Machine machine : machines) {
+            machine.doWork();
```

Comment:
The messages within `doWork` and `stopWork` methods should be informative and indicate the type of machine that is working. Ensure that the implementations of these methods in the respective classes follow this guideline.